packages/@o3r/design/src/core/design-token/renderers/design-token-style.renderer.ts

```diff
@@ -80,6 +98,31 @@ export const getTokenSorterByRef: TokenListTransform = (variableSet) => {
   };
 };
 
+/**
+ * Retrieve default file writer (based on Node `fs.promise.writeFile` interface)
+ * @param existsFile Function determining if the file exists
+ * @param logger Custom Logger
+ * @returns
+ */
+export const getDefaultFileWriter = (existsFile: (file: string) => boolean, logger?: Logger): typeof fs.writeFile => {
+  return async (file, ...args) => {
+    // eslint-disable-next-line @typescript-eslint/no-base-to-string
```

Comment:
With the new ESLint config that we prepare we should add a comment to explain why we disable a rule following the format:
`// eslint-disable-next-line <rule-name> -- <explication>`